### ISSUE TITLE
With Ratio Locked - Add edge-like responses 

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#247]
Because a learner cannot hit the bottom when the hands are locked in ratio, we might need some special responses to indicate they should change directions to continue exploring.

Here is a draft idea for a Context Response when a hand reaches either the lowest possible position or the highest possible position when the hands are locked in ratio.
- {{Right hand}}, {{at top}}. Move {{down}} to explore more.
- {{Left hand}}, {{near bottom}}. Move {{up}} to explore more.

I'm not 100% sure we need these responses, though I think we might.

I'd like to wait and see after #245 is implemented before making a final decision on if we need these, and what exactly the wording might be.

**Comment by terracoda** [comment]
@zepumph, tagging you so you are aware of this issue, but no need to implement until after I test changes coming in #245.

**Comment by terracoda** [comment]
@zepumph, actually, there is not need to wait on this. Assigning over to you for implementation.

**Comment by terracoda** [comment]
I imagine, there could be cases like this:
- {{Right hand}}, {{at top}}. Move {{down}} to explore more.
- {{Right hand}}, {{near top}}. Move {{down}} to explore more.
- {{Left hand}}, {{near bottom}}. Move {{up}} to explore more.
- {{Left hand}}, {{at top}}. Move {{down}} to explore more.
- {{Left hand}}, {{near top}}. Move {{down}} to explore more.
- {{Right hand}}, {{near bottom}}. Move {{up}} to explore more.

**Comment by terracoda** [comment]
Maybe I can make this simpler, basically, I just want 2, maybe four possibilities, a max and min point when ratio is locked.

**Comment by terracoda** [comment]
Oh, it seems regardless of teh set challenge, the max will always be 10/ at top and the min will always be 0.5 / near bottom, so going back to the same plan. I think there are only 4 possibilities.

 - {{Right hand}}, {{at top}}. Move {{down}} to explore more.
- {{Left hand}}, {{near bottom}}. Move {{up}} to explore more.
- {{Left hand}}, {{at top}}. Move {{down}} to explore more.
- {{Right hand}}, {{near bottom}}. Move {{up}} to explore more.
    {{Right hand}}, {{near bottom}}. Move {{up}} to explore more.

**Comment by terracoda** [comment]
I added "hands".
Ok, ready for you now @zepumph. Apologies for my last minute iterations.

**Comment by zepumph** [comment]
Implemented above. I like that it is based on what hand is at the extremity, and not what hand was just moved. Please review.

**Comment by zepumph** [comment]
Today in design meeting, we decided that we only want this if they try to "go beyond" the edge, not on first getting to the min/max edge. I'll investigate!

**Comment by terracoda** [comment]
@zepumph, we did these "go beyond edge" responses in GFL & GFLB.

**Comment by terracoda** [comment]
@KatieWoe mentioned in #317. I will mark that issue as a duplicate issue as we already planning on updating the edge alert to be a go-beyond-edge alert.

**Comment by zepumph** [comment]
The above adds support to only use edge alerts when trying to move past the edge, not when arriving. @terracoda please review.

**Comment by terracoda** [comment]
These responses are working great!
Thanks!
